src/github/mod.rs

```diff
@@ -189,6 +196,48 @@ impl SyncGitHub {
         Ok(diffs)
     }
 
+    fn diff_toml_teams(&self) -> anyhow::Result<Vec<OrgMembershipDiff>> {
```

Comment:
I think that it would be simpler and more robust implementation would be to do this instead:
1) Go through all teams and gather a set of all team members, for each org that we find (so essentially a `HashMap<Org, HashSet<u64>>`)
2) Go through each org, and do a set difference between its members and the team members found in 1). Users that come out of this operation (those that are not in the team) should be stored in the `OrgMembershipDiff`